### PR TITLE
Add basic options proto to support adding the fields and message tags options

### PR DESCRIPTION
**Pull request by sadlil**

### CODE REVIEW

File: protobuf/option.proto [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  // size specifies column size, default 255
+  int32 size = 10000003;
+  // primary_key specifies column as primary key
+  bool primary_key = 10000004;
+  // unique specifies column as unique
+  bool unique = 10000005;
+  // default specifies column default value
+  string default = 10000006;
+  // not_null specifies column as NOT NULL
+  bool not_null = 10000007;
```
Comment (mirshahriar) [inline protobuf/option.proto]: what do you think about `allow_null` or `is_nullable`?

**Comment by mirshahriar** [comment]
Did you try using `table_name` from **MessageOptions** in a Message?

**Comment by sadlil** [comment]
@aerokite added an example, and revised the options. you can just run `make compile` to compile everything.

**Review by alaminopu — APPROVED** [review]
Looks good to me.

**Review by s4kibs4mi — APPROVED** [review]
Seems fine.

**Comment by sadlil** [comment]
@aerokite any thoughts on this?

**Comment by mirshahriar** [comment]
LGTM. I have checked this far.

**Review by mirshahriar — APPROVED** [review]

**Pull request merged**